cpukit/posix/src/pthread.c

```diff
@@ -89,15 +89,16 @@ static bool _POSIX_Threads_Sporadic_budget_TSR_filter(
   void             *arg
 )
 {
-  the_thread->real_priority = *new_priority;
+  the_thread->Priority_node.real_priority = *new_priority;
 
   /*
    * If holding a resource, then do not change it.
    *
    * If this would make them less important, then do not change it.
    */
   return !_Thread_Owns_resources( the_thread ) &&
-    _Thread_Priority_less_than( the_thread->current_priority, *new_priority );
+    _Thread_Priority_less_than( the_thread->Priority_node.current_priority,
+            *new_priority );
```

Comment:
Wrong line wrapping. There should not be anything after the opening parens on the first line, and the closing parens should be aligned with the first line. e.g.,:
```C
    _Thread_Priority_less_than(
        the_thread->Priority_node.current_priority,
        *new_priority
    );
```
Fix this everywhere.